docs/source/models.rst

```diff
@@ -127,6 +128,13 @@ For environment recreation, we automatically log ``conda.yaml`` and ``requiremen
 
 conda.yaml
     When saving a model, MLflow provides the option to pass in a conda environment parameter that can contain dependencies used by the model. If no conda environment is provided, a default environment is created based on the flavor of the model. This conda environment is then saved in ``conda.yaml``.
+python_env.yaml
+    This file contains the following information that's required to restore a model environment using virtualenv.
+
+    - Python version
+    - Pip requirements for ``pip``, ``setuptools``, and ``wheel``
+    - Pip requirements of the model (reference to ``requirements.txt``)
+
 requirements.txt
     The requirements file is created from the `pip portion <https://www.anaconda.com/blog/using-pip-in-a-conda-environment>`_ of the ``conda.yaml`` environment specification. Additional pip dependencies can be added to ``requirements.txt`` by including them as a pip dependency in a conda environment and logging the model with the environment. 
```

Comment:
`Additional pip dependencies can be added to ``requirements.txt`` by including them as a pip dependency in a conda environment and logging the model with the environment`

Can we also update this to tell users that they can add new pip reqs using the `pip_requirements` arg?

Reply:
Sure, what do you think about this?

```diff
diff --git a/docs/source/models.rst b/docs/source/models.rst
index 9727ca1e5..b65f66bd2 100644
--- a/docs/source/models.rst
+++ b/docs/source/models.rst
@@ -136,7 +136,7 @@ python_env.yaml
     - Pip requirements of the model (reference to ``requirements.txt``)
 
 requirements.txt
-    The requirements file is created from the `pip portion <https://www.anaconda.com/blog/using-pip-in-a-conda-environment>`_ of the ``conda.yaml`` environment specification. Additional pip dependencies can be added to ``requirements.txt`` by including them as a pip dependency in a conda environment and logging the model with the environment. 
+    The requirements file is created from the `pip portion <https://www.anaconda.com/blog/using-pip-in-a-conda-environment>`_ of the ``conda.yaml`` environment specification. Additional pip dependencies can be added to ``requirements.txt`` by including them as a pip dependency in a conda environment and logging the model with the environment or using the ``pip_requirements`` argument of the `mlflow.<flavor>.log_model` API.
```

Reply:
Perfect!